airflow/providers/google/cloud/hooks/mlengine.py

```diff
@@ -65,13 +65,13 @@ def _poll_with_exponential_delay(
                 log.info("Operation is done: %s", response)
                 return response
 
-            time.sleep((2**i) + (random.randint(0, 1000) / 1000))
+            time.sleep((2**i) + random.random())
```

Comment:
This is ever so slightly different—the `randint` version can produce exactly 1, while `random` can’t. I don’t know if this would be relevant though.

Reply:
Correct, but this goes directly to `time.sleep`, so I don't think that difference matters.